solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -265,10 +281,30 @@ public MiniSolrCloudCluster(int numServers, Path baseDir, String solrXml, JettyC
     solrClient = buildSolrClient();
   }
 
-  private void waitForAllNodes(int numServers, int timeout) throws IOException, InterruptedException {
+  private void waitForAllNodes(int numServers, int timeoutSeconds) throws IOException, InterruptedException {
```

Comment:
Some Javadocs that we could add here :

"This method wait till all Solr JVMs ( Jettys ) are running . It waits upto the timeout (in seconds) for the JVMs to be up before throwing IllegalStateException"

Reply:
Cool, I'll add, as well as a note about how you don't need to use this after starting the cluster with the builder - just when adding new jetties.